.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           key: gradle-cache-{{ checksum "manage-accounts-specs/build.gradle" }}
       - run:
           name: Run Integration Tests
-          command: ./gradlew build
+          command: ./gradlew test
```

Comment:
no idea why we were trying to `build` here

Reply:
Should this be check?